src/components/NewCard/README.md

```diff
@@ -0,0 +1,4 @@
+## NewCard
+An implementation of an NewCard for TPAs
+This Readme was auto-generated and needs to be filled manually
+
```

Comment:
We should add a short description on how to use the card.
no need to add the stylable variables table and all that.